operators/src/processing/expression/mod.rs

```diff
@@ -284,6 +283,7 @@ impl RasterOperator for Expression {
             sources,
             expression,
             map_no_data: self.params.map_no_data,
+            output_no_data_value: self.params.output_no_data_value,
```

Comment:
why is this needed now? Can't we just put 0 there? I guess we really have to think about what the meaning of a pixel value is if it is masked as no data.

Reply:
i answered that above